notebooks/238-attention-center/238-attention-center.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>here you use AUTO, but above CPU, could you please align?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1044/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
done